src/components/App/index.js

```diff
@@ -79,30 +93,35 @@ export const App = ({ options }) => {
     quiz = new GameEngine(gameMode, apiDataFetcher, handleGameOver);
     const gameView = new GameView(handleAnswerSelected);
     try {
+      handleLoaderDisplay();
       await quiz.fetchAllQuestionsForMode(gameMode);
+      handleLoaderDisplay();
```

Comment:
to też jest zjebane. chciałam to wrzucić w ApiDataFetcher w metodzie getAllDataForGameMode w czymś na zasadzie:
`if([...Array(pagesToDownload)].length - 1 === indx) { this.handleLoaderDisplay()}` no ale nie szło, nie wiem trochę dlaczego